pyvo/dal/tap.py

```diff
@@ -149,6 +149,38 @@ def tables(self):
                 vosi.parse_tables(response.raw.read), tables_url)
         return self._tables
 
+    def _parse_examples(self, examples_uri, /, depth=0):
+        """returns the TAP queries from a DALI examples URI.
+        """
+        if depth > 5:
+            raise Exception("Suspecting endless recursion when"
```

Comment:
Please us a more specific Exception. Is this e.g. a `ValueError`, or maybe just a warning?